float32[] VFH_data
